libs/lua-simple-uci/Makefile

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (C) 2015 Nils Schneider <nils@nilsschneider.net>
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=simple-json
```

Comment:
Also, the directory is called lua-simple-uci, but everywhere else this is just called simple-uci... I prefer the version with prefix or suffix, so it is clear that it is a Lua library.